packages/web-components/src/player-component/player.class.ts

```diff
@@ -153,6 +155,8 @@ export class PlayerWrapper {
         this.player?.removeEventListener('trackschanged', this.onTrackChange.bind(this));
 
         this.player?.removeEventListener('emsg', this.onShakaMetadata.bind(this));
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        (this.player as any).removeEventListener('wallclock', this.onWallclock.bind(this));
```

Comment:
Can you run lint? I suspect this is going to fail, and if you check in a lint-breaker, the validation build won't catch it, and I will have to deal with it.

Reply:
There is suppression on the line above.